lesson-6/JAMROZ-week 6/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Mock Portfolio Site</title>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:400,700" rel="stylesheet">
+  </head>
+  <body>
+    <nav>
+      <h3>My Portfolio Site</h3>
+      <ul>
+        <li><a href="#">MY WORK</a></li>
+        <li><a href="#">ABOUT ME</a></li>
+        <li><a href="#">CONTACT ME</a></li>
+
+      </ul>
+    </nav>
+
+    <header class="featured-project container">
+      <div>
+        <h1>My Featured Project</h1>
+        <time>February 1, 2018</time>
+        <p>This  is  the  coolest  project  I’ve  ever  done.  It  showcases  many  of  my  awesome  skills.  I  can  code  and  I  can  make  stuff.  Click  on  this  button  below  to  read  more  about  my  sweet  project.</p>
+        <a href="#">READ MORE</a>
+           <style>
+            a:link {
+              color:#FC6062;
+              text-decoration: none;
+            }
+          </style>
+      </div>
+      <img src="images/pic_placehold_lrg.png" />
+
+
+    </header>
+
+    <header>
+      <div>
+      <h3>Another Cool Project</h3>
+      <time>February 1, 2018</time>
+      <p>This is another project I've done that's not as important as the big thing I want to show, but still cool.</p>
+      <a href="#">READ MORE</a>
+      </div>
+      <img src="images/pic_placehold_sm.png" />
+
+    </header>
+
+    <header>
```

Comment:
This is not wrong, but usually there is only one `header` per page (think about just talking about a web page, it would be strange to call three sections `header`, right?)